### PR TITLE
vorbis-tools: Fix build on Ventura

### PR DESCRIPTION
**Pull request by iMichka**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/vorbis-tools.rb [hunk 1]
```diff
@@ -25,16 +25,27 @@ class VorbisTools < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "b137af1a327bf93e3922244bf03ab2eef1151838483b2e71e1e35c5254d32cc4"
   end
 
+  # macOS `ld64` does not like the `.la` files created during the build.
+  depends_on "llvm" => :build if DevelopmentTools.clang_build_version >= 1400
```
Comment (carlocab) [inline Formula/vorbis-tools.rb:29]: We should report this upstream. This most likely means some non-object files are being included in `.a` archives by mistake.

**Comment by carlocab** [comment]
Let's try a different fix...

**Comment by iMichka** [comment]
Trying again, I added libtool to the build to make it pass.

**Comment by carlocab** [comment]
Cool, thanks. Will upstream the patch then we can get this merged. Might not get to it till later though.

**Comment by iMichka** [comment]
@carlocab still want to upstream the patch? Looking at https://github.com/xiph/vorbis-tools/issues/27 it's unlikely this will get a new release anyway.

**Comment by carlocab** [comment]
...ah. Guess not then.

**Review by carlocab — APPROVED** [review]

**Pull request merged**